test/draw_test.py

```diff
@@ -1,7 +1,325 @@
-import unittest
-from pygame.tests import test_utils
+#################################### IMPORTS ###################################
+
+if __name__ == '__main__':
```

Comment:
This is_pygame_pkg stuff should be removed.

The imports should be something like:
```python
import unittest

import pygame
from pygame import draw
from pygame.locals import SRCALPHA
from pygame.tests import test_utils
```